crt_portal/cts_forms/templates/vot_resources.html

```diff
@@ -0,0 +1,111 @@
+              <a href="#civilrights-violation">{% trans "Report a possible civil rights violation" %}</a>
+            </li>
+            <li class="usa-sidenav__item">
+              <a href="#voting-rights">{% trans "Learn about my voting rights" %}</a>
+            </li>
+          </ul>
+        </ul>
+      </div>
+      <div class="tablet:grid-col-8">
+        <h1 id="get-help">
+          {% trans "Report voting issues" %}
+        </h1>
+        <div class="crt-landing--separator"></div>
+
+        <section id="criminal-violation">
+          <h2>{% trans "Report a possible federal criminal violation" %}</h2>
+          
+          <p>
+            {% trans "To report a possible federal crime, go to" %} <a class="external-link--blue" href="https://tips.fbi.gov">tips.fbi.gov</a>  {% trans " or call 800-CALL-FBI." %}
+          </p>
+          <p>
+            {% trans "EXAMPLES" %}
+          </p>
+          <ul>
+            <li>{% trans "Threat against voters" %}</li>
+            <li>{% trans "Threat against election officials" %}</li>
+            <li>{% trans "Election fraud" %}</li>
+          </ul>
+        </section>
+        <section id="polling-place">
+          <h2>{% trans "Find polling place location and hours" %}</h2>
+
+          <p>
+            {% trans "Go to" %} <a class="external-link--blue" href="https://usa.gov/election-office">usa.gov/election-office</a>  {% trans " to find your state election website." %}
+          </p>
+        </section>
+        <section id="civilrights-violation">
+          <h2>{% trans "Report a possible civil rights violation" %}</h2>
+
+          <p>
+            {% trans "To report civil rights violations, go to " %} <a class="link--blue" href="https://civilrights.justice.gov">civilrights.justice.gov</a>.
+          </p>
+
+          <p>
+            {% trans "EXAMPLES" %}
+          </p>
+          <ul>
+            <li>{% trans "Discrimination on the basis of race, color, or membership in a language minority gorup while voting or attempting to vote;" %}</li>
+            <li>{% trans "Voter registration issues;" %}</li>
+            <li>{% trans "Denial or lack of accessibility while voting;" %}</li>
+            <li>{% trans "Military or overseas absentee voting issues." %}</li>
+          </ul>
+        </section>
+        <section id="voting-rights">
+          <h2>{% trans "Learn about my voting rights" %}</h2>
+
+          <p>
+            <ul>
+            <li>{% trans "Want to know your federal rights? Download the Justice Department's new " %}<a class="link--blue" href="https://www.justice.gov/file/1517936/download">Know Your Voting Rights</a> guide.
+            </li>
+            <li>{% trans "Do you have a criminal conviction? Wondering if you can vote in your state?<br>Download the Justice Department's new " %}<a class="link--blue" href="https://www.justice.gov/voting/file/1507306/download">Guide to State Voting Rights that Apply After Criminal Conviction</a></li>
```

Comment:
Should "Guide to State Voting Rights that Apply After Criminal Conviction" be translated?